### ISSUE TITLE
[A11Y] App rotation

### ISSUE THREAD

**Issue by erikkroes** [Tiqr/eduid-app-android#264]
## Finding and problem

The app cannot be rotated to landscape. Not everybody has the freedom to orient their device whatever way they want. Restricting orientation to portrait can exclude people.

## Solution

Make the app rotate-able. This can be as simple and "design-less" as with the ipad-app.

## Resources
- https://www.w3.org/WAI/WCAG22/Understanding/orientation.html

**Comment by Tyskai** [comment]
We, PM en designers, zetten hem even in de icebox en kijken later
